src/app/components/Search/Search.jsx

```diff
@@ -124,48 +117,41 @@ class Search extends React.Component {
 
   render() {
     return (
-
-      <SearchBar
-        id="mainContent"
-        onSubmit={this.triggerSubmit}
-        className="content-primary"
-        attributes={{
-          method: 'POST',
-          action: `${appConfig.baseUrl}/search`,
-        }}
-      >
-        <div id="search-container">
-          <Select
-            id="search-by-field"
-            onChange={this.onFieldChange}
-            selectedOption={this.state.field}
-            name="search_scope"
-            >
-            <option value="all">All fields</option>
-            <option value="title">Title</option>
-            <option value="journal_title">Journal Title</option>
-            <option value="contributor">Author/Contributor</option>
-            <option value="standard_number">Standard Numbers</option>
-            <option value="subject">Subject</option>
-          </Select>
-          <Input
-            type="text"
-            id="search-query"
-            aria-label="Search by keyword, title, journal title, or author/contributor"
-            aria-controls="results-description"
-            placeholder="Keyword, title, journal title, or author/contributor"
-            onChange={this.inputChange}
-            value={this.state.searchKeywords}
-            name="q"
-          />
-          <SearchButton
-            onClick={this.submitSearchRequest}
-          />
-        </div>
+      <>
+        <SearchBar
+          buttonOnClick={this.submitSearchRequest}
+          id="mainContent"
+          onSubmit={this.triggerSubmit}
+          className="content-primary"
+          method='POST'
+          action={`${appConfig.baseUrl}/search`}
+          labelText="SearchBar Label"
+          selectProps={{
+            labelText: 'Select a category',
+            name: 'search_scope',
+            optionsData: [
+              "All fields",
+              "Title",
+              "Journal Title",
+              "Author/Contributor",
+              "Standard Numbers",
+              "Subject",
```

Comment:
I believe this needs to be an array of hashes with both text label and option value, like:
```
optionsData: [
  { text: "All fields", value: "default" },
  { text: "Title", value: "title" },
  ...
]
```
As it is, the control is using the option label as the `&search_scope` param, when the discovery-api actually expects [a different set of values](https://github.com/NYPL/discovery-api/blob/main/swagger.v1.1.x.json#L86-L92). E.g. right now, searches [do this](https://qa-www.nypl.org/research/research-catalog/search?q=33433113614832&search_scope=Standard%20Numbers), but they should [do this](https://qa-www.nypl.org/research/research-catalog/search?q=33433113614832&search_scope=standard_number)

Reply:
Yes, you're right. It's a limitation of the DS which we didn't know until this use case. An update was merged in the DS https://github.com/NYPL/nypl-design-system/pull/1095 but this might not go out until next week. We can push the DS release with this fix earlier if we want this update to go out. Otherwise, if we can wait until next week, I'll make a PR against this with the update after we release (on 8/4).